### DISCUSSION TITLE
我准备提交一些新功能，我应该为每个组件单独创建一个 pr 吗？

### DISCUSSION THREAD

**Discussion by bluesky335** [youzan/vant#12739] [Q&A]
我准备为这个 issue: https://github.com/youzan/vant/issues/12711 提交代码。

涉及到 3 个组件，PickerGroup、DatePicker、TimePicker

做了以下变更：

1、为他们三个添加了 confirm 方法
2、为 DatePicker、TimePicker 添加了获取当前选择值的方法
3、为 PickerGroup 添加了隐藏 toolbar 的 props。

我是第一次为 vant 提交代码。
我应该为三个组件分别提交 pr 还是说 按照上面列出的功能 分别提交 3 个 pr 还是直接全部提交 1 个 pr。

**Comment by chenjiahan** [comment]
感谢贡献❤️

建议是分开创建 PR，这样有利于 review 和生成 changelog
